engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateMigrationService.java

```diff
@@ -469,8 +538,12 @@ private String loadResourceTemplate(String name, String extension) {
      */
     private void createInstantEmailTemplate(List<String> warnings, String bundleName, String appName, List<String> eventTypeNames,
             String subjectTemplateName, String subjectTemplateExtension, String subjectTemplateDescription,
-            String bodyTemplateName, String bodyTemplateExtension, String bodyTemplateDescription) {
+            String bodyTemplateName, String bodyTemplateExtension, String bodyTemplateDescription, boolean useTemplateV2) {
 
+        if (useTemplateV2) {
+            subjectTemplateName += "V2";
+            bodyTemplateName += "V2";
+        }
```

Comment:
Is there something preventing this?
```suggestion
        if (featureFlipper.isRhosakEmailTemplatesV2Enabled()) {
            subjectTemplateName += "V2";
            bodyTemplateName += "V2";
        }
```
If not, I would really like that change to be applied as it would make the PR much lighter.

Reply:
This method is always called with the activation flag like `featureFlipper.isRhosakEmailTemplatesV2Enabled()` at last parameter ie : 
`        createInstantEmailTemplate(
                warnings, "application-services", "rhosak", List.of("scheduled-upgrade"),
                "Rhosak/scheduledUpgradeTitle", "txt", "Rhosak scheduled upgrade email title",
                "Rhosak/scheduledUpgradeBody", "html", "Rhosak scheduled upgrade email body",
                featureFlipper.isRhosakEmailTemplatesV2Enabled()
        );`
This is why we don't need to check this flag one more time here. WDYT ?

Reply:
I asked a stupid question, please ignore it :smile: